website/docs/docs/build/unit-tests.md

```diff
@@ -219,10 +216,19 @@ dbt test --select test_is_valid_email_address
 
 Your model is now ready for production! Adding this unit test helped catch an issue with the SQL logic _before_ you materialized `dim_customers` in your warehouse and will better ensure the reliability of this model in the future. 
 
-
 ## Unit testing incremental models
 
-When configuring your unit test, you can override the output of macros, vars, or environment variables. This enables you to unit test your incremental models in "full refresh" and "incremental" modes. 
+When configuring your unit test, you can override the output of macros, vars, or environment variables. This enables you to unit test your incremental models in "full refresh" and "incremental" modes.
+
+:::tip
```

Comment:
```suggestion
:::warning
```

Reply:
ah, i prefer a tip than warning for this. but how about a :::note::: to match the earlier callout that basically says the same thing
![Screenshot 2025-01-13 at 17 14 17](https://github.com/user-attachments/assets/f2de063b-ad75-425f-a360-b36eaf5c1e16)